operations/configuration_management/NCN_Node_Personalization.md

```diff
@@ -27,17 +27,19 @@ The following procedure describes how to correctly edit the `sat bootprep` files
     - name: ncn-personalization
     ```
 
-1. (`ncn-m#`) Acquire a copy of the current CPE and Analytics products CFS configuration values already in use.
- Obtain the values for the `cloneUrl`, `commit`, and `playbook` lines for each of those two layers in the next step to ensure that they are personalized with the desired configuration for CPE and Analytics.
+1. In the case of an upgrade, execute the following two substeps:
 
-    ```bash
-    cray cfs components describe <ncn-xname> --format json
-    ```
+    1. (`ncn-m#`) Acquire a copy of the current CPE and Analytics products CFS configuration values already in use.
+     Obtain the values for the `cloneUrl`, `commit`, and `playbook` lines for each of those two layers in the next step to ensure that they are personalized with the desired configuration for CPE and Analytics.
+
+        ```bash
+        cray cfs components describe <ncn-xname> --format json
+        ```
 
-1. (`ncn-m#`) Edit the `management-bootprep-node-personalization.yaml` file to replace the CPE and Analytics layer with the playbook, commit hash, and product values already in use on the NCNs for CPE.
-This must be done because the new version of CPE and Analytics has not yet been installed at this time in the upgrade procedure.
+    1. (`ncn-m#`) Edit the `management-bootprep-node-personalization.yaml` file to replace the CPE and Analytics layer with the playbook, commit hash, and product values already in use on the NCNs for CPE.
+    This must be done because the new version of CPE and Analytics has not yet been installed at this time in the upgrade procedure.
```

Comment:
```suggestion
    This must be done because the new versions of CPE and Analytics have not yet been installed at this time in the upgrade procedure.
```

Reply:
Fixed, thanks